saleor/product/models.py

```diff
@@ -35,9 +34,11 @@ class Category(MPTTModel):
     description = models.TextField(
         pgettext_lazy('Category field', 'description'), blank=True)
     parent = models.ForeignKey(
-        'self', null=True, blank=True, related_name='children',
-        verbose_name=pgettext_lazy('Category field', 'parent'),
-        on_delete=models.CASCADE)
+        'self',
+        null=True,
+        blank=True,
+        related_name='children',
+        verbose_name=pgettext_lazy('Category field', 'parent'))
```

Comment:
I understand this kind of formatting the arguments but it's not consistent with our codebase.